### ISSUE TITLE
Connecting a manasteel cable to a compact machine crashes the game

### ISSUE THREAD

**Issue by CreationPro** [FTBTeam/FTB-Modpack-Issues#2672]
### Modpack

FTB Interactions

### Modpack version

2.12.1

### Log Files

[crash-2023-02-06_10.19.12-server.txt](https://github.com/FTBTeam/FTB-Modpack-Issues/files/10663198/crash-2023-02-06_10.19.12-server.txt)

### Describe the bug

Attaching a zero-loss GTCE cable to a compact machine crashes the server (permanently).

From what I could gather, it sends the server into an infinite loop of ticks.

How I got the issue is I attached a manasteel cable to my power outputs (MV turbines), placed down a compact machine (max size), put a tunnel on the inside facing the cable and placed a cable on the inside, connecting the outside and the inside. As a result, the server crashed.

### Steps to reproduce

1. Place down a GTCE generator (MV turbine)
2. Place down a compact machine.
3. Connect the cable to the compact machine.
4. Place a tunnel on the inside of the compact machine, let it face the cable.
5. Place a cable on the inside of the machine next to the tunnel.
6. Power up the turbine.

### Expected behaviour

I expected the cable to provide power to the GTCE machines inside. Instead, I got a crash.

### Additional information

I didn't place any machinery inside the compact machine, the only things I placed was the tunnel and a cable. Upon connecting outside and inside, the server crashed. The cable on the outside was powered at the time.

Relevant issues I could find: https://github.com/FTBTeam/FTB-Interactions/issues/1322
https://github.com/FTBTeam/FTB-Interactions/issues/1191
https://github.com/GregTechCE/GregTech/issues/1374

If this can't be fixed on our side but rather needs to be fixed on mod makers', perhaps put a warning in the questbook not to connect EU cables to compact machines as a stopgap solution?

**Comment by Saereth** [comment]
Yeah dont do that, it's been reported to both compact machines and GT and neither are fixing it. I will definitely go add a warning to the questbook, good call

**Comment by CreationPro** [comment]
> Yeah dont do that, it's been reported to both compact machines and GT and neither are fixing it. I will definitely go add a warning to the questbook, good call

Good enough, it's the same kind of deal as with dolly + crate interaction. We just don't do it.
Thank you!

**Comment by HaoSs07** [comment]
> Yeah dont do that, it's been reported to both compact machines and GT and neither are fixing it. I will definitely go add a warning to the questbook, good call

I think you forgot something.. gues how I ended up here.

**Comment by Gaz492** [comment]
Thank you for creating this issue/suggestion, however this modpack is now considered a legacy pack and is therefore no longer receiving any bug fix updates or mod updates.
Any updates to this pack will only be released if there is a security issue.

We may have something up our sleeves, some may call it a remaster, keep an eye out on our social media platforms for more information.
